src/gateway/tasks/loadPeers.ts

```diff
@@ -16,6 +16,9 @@ export async function runLoadPeersTask(context: Application.BaseContext) {
 
   logger.info("Starting [loadPeers] task.");
 
+  // Hmm, why not use `setInterval` instead?
```

Comment:
the reason is described here - https://javascript.info/settimeout-setinterval#nested-settimeout
and here https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Timeouts_and_intervals#recursive_timeouts .

I want to have a fixed delay between consecutive calls to arweave.net gateway and peers.

Reply:
Bbetween calls completion, right?

Reply:
yup, exactly. As the response times from both the gateway and peers fluctuate heavily I want to be sure that there's at least some time between the calls (in order to not get rate limitted, etc.)